### PR TITLE
Add a couple of missing options

### PR DESCRIPTION
**Pull request by Daimona**

### CODE REVIEW

**Comment by Daimona** [comment]
I've also changed the value for SPACE_BEFORE_COLON_IN_RETURN_TYPE, now that we have [rules for it](https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP#Spaces). @reedy Any chance you could take a look at this?

**Comment by vedmaka** [comment]
Would be nice to have this merged

**Comment by reedy** [comment]
Merging doesn't help when I don't have a working environment to build the plug-in...

File: src/org/mediawiki/MediaWikiPhpPredefinedCodeStyle.java [hunk 1]
```diff
@@ -42,7 +42,8 @@ public void apply(CodeStyleSettings settings) {
         commonSettings.SPACE_WITHIN_CATCH_PARENTHESES = true;
         commonSettings.SPACE_WITHIN_SWITCH_PARENTHESES = true;
         commonSettings.SPACE_WITHIN_ARRAY_INITIALIZER_BRACES = true;
-        commonSettings.SPACE_AFTER_TYPE_CAST = true;
+        commonSettings.SPACE_AFTER_TYPE_CAST = false;
+        commonSettings.SPACES_AROUND_VAR_WITHIN_BRACKETS = false;
```
Comment (reedy) [inline src/org/mediawiki/MediaWikiPhpPredefinedCodeStyle.java:46]: Looks like this one actually upsets it at compile... Commeting it out for now in my rebuild

Comment (reedy) [reply src/org/mediawiki/MediaWikiPhpPredefinedCodeStyle.java:46]: fixed by https://github.com/reedy/phpstorm-plugin-mediawiki/commit/9c45c6f15e6b7dd53d12858d70a8163da218a807, I think...

**Pull request merged**